compose/ui/ui-test-junit4/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -261,6 +266,16 @@ class SkikoComposeUiTest(
         return surface.makeImageSnapshot()
     }
 
+    fun SemanticsNodeInteraction.captureToImage(): ImageBitmap {
```

Comment:
API change: new method

Reply:
nit: Can we use `waitForIdle()` + `surface.makeImageSnapshot(iRect)` for better performance?